modules/javafx.controls/src/main/resources/com/sun/javafx/scene/control/skin/modena/modena.css

```diff
@@ -377,6 +377,9 @@ is being used to size a border should also be in pixels.
      **************************************************************************/
 
     -fx-background-color: -fx-background;
+
+    /* avoid race condition during initialization */
+    -fx-table-cell-border-color: transparent;
```

Comment:
A bit of a nitpick, but this should be next to the other cell colors, so in line 360 under `-fx-cell-focus-inner-border`. 
Also, we should probably add a description to the color like the other variables have here instead of explaining why it is here. :)

Reply:
1) good point, let's move it there
2) the color name is self-explanatory, but why it is there is not.  Perhaps I should also include the ticket number for reference?